components/footer/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { inject, observer } from 'mobx-react'
+import format from 'date-fns/format'
+
+const Footer = inject('store')(
+  observer((props) => {
+    const { months } = props.store
+    return (
+      <div>
+        <footer className="footer">
+          {months.map((month, index) => (
+            <button
+              onClick={() => props.store.setSelectedMonth(month.monthId)}
+              className={month.monthId === props.store.selectedMonth ? 'buttonSelected' : 'button'}
```

Comment:
In order to not have to duplicate CSS declarations (saves a lot time when you want to change something), just try this:
```suggestion
              className={month.monthId === props.store.selectedMonth ? 'button selected' : 'button'}
```

and then declare a class `.button`, which has all the styles for the button, and then class `.selected` which only has the changes you want to apply to it when it's selected (i think you change background color and font weight).